src/main/java/axoloti/utils/Preferences.java

```diff
@@ -200,6 +208,37 @@ public String getCurrentFileDirectory() {
         return CurrentFileDirectory;
     }
 
+    public IPlatform getPlatform(Patch p) {
+        IPlatform target = null;
+        if (Platform != null && !Platform.isEmpty()) {
+            try {
+                Logger.getLogger(Preferences.class
+                        .getName()).log(Level.INFO, "Attempting to load target {0}", Platform);
+                ClassLoader cl = Axoloti.class.getClassLoader();
+                Class c = cl.loadClass(Platform);
+                Object o = c.newInstance();
```

Comment:
Again, this seems like a highly unlikely scenario, distributing a separate platform JAR I mean. We're just overengineering in the present for something that may not exist in the future. I think that should really be avoided at all costs. Yeah, I wasn't suggesting that you necessarily include it in preferences, just that there is the potential that something like that would exist in the future.